app/components/public/ticket-list.js

```diff
@@ -29,14 +29,20 @@ export default Component.extend(FormMixin, {
   }),
   hasTicketsInOrder: computed('tickets.@each.orderQuantity', function() {
     return sumBy(this.tickets.toArray(),
-      ticket => ticket.getWithDefault('orderQuantity', 0)
+      ticket => (ticket.orderQuantity || 0)
     ) > 0;
   }),
 
   total: computed('tickets.@each.orderQuantity', 'tickets.@each.discount', function() {
-    return sumBy(this.tickets.toArray(),
-      ticket => (ticket.getWithDefault('price', 0) - ticket.getWithDefault('discount', 0)) * ticket.getWithDefault('orderQuantity', 0)
-    );
```

Comment:
Why are you replacing getWithDefault?

Reply:
@CosmicCoder96 Please see this : https://github.com/fossasia/open-event-frontend/pull/3067#discussion_r290427205